### ISSUE TITLE
Clipping of the bottom portion of the screen on mobile

### ISSUE THREAD

**Issue by ri0nardo** [waterthetrees/wtt_front#459]

**Comment by Dawoodeveloper** [comment]
Can i solve this issue?
Please assign this issue to me. so i will contribute in this issue.

**Comment by ri0nardo** [comment]
@Dawoodeveloper are you still interested in this ticket?

**Comment by Dawoodeveloper** [comment]
Yes, I'm interested to contribute in your project. Allow me to do

**Comment by Dawoodeveloper** [comment]
I will work on your project after 20th of April.

**Comment by ri0nardo** [comment]
@Daudsarfraz hows progress? Any updates?

**Comment by Daudsarfraz** [comment]
I'm really sorry, i'm a 6th semester student. I have also my semester projects. That's why i can't complete your project on time. Now i have final exams from 24-may to 5-june.so after that i will windup your project. Thanks

**Comment by ri0nardo** [comment]
@Daudsarfraz any updates on this ticket?
